spec/models/artist_spec.rb

```diff
@@ -1,5 +1,34 @@
 require 'rails_helper'
 
 RSpec.describe Artist, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+    describe 'Artist' do
+     let(:attributes) {{
+         name: "Lady Gaga",
+     }}
+
+     it 'inherits from ActiveRecord::Base' do
+       expect(Artist.superclass).to eq(ActiveRecord::Base)
+     end
+
+   end
+
+     context 'Artist.new' do
+       let(:artist) { Artist.new }
+       it 'has a name' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.